src/main/java/ch/njol/skript/variables/VariablesStorage.java

```diff
@@ -224,9 +225,24 @@ public final boolean load(SectionNode sectionNode) {
 			// Set the backup interval, if present & enabled
 			if (!"0".equals(getValue(sectionNode, "backup interval"))) {
 				Timespan backupInterval = getValue(sectionNode, "backup interval", Timespan.class);
-
+				int toKeep = getValue(sectionNode, "backups to keep", Integer.class);
+				boolean removeBackups = false;
+				boolean startBackup = true;
 				if (backupInterval != null)
-					startBackupTask(backupInterval);
+					if (toKeep == 0) {
+						startBackup = false;
+					} else if (toKeep >= 1) {
+						removeBackups = true;
+					}
+					if (startBackup) {
+                        startBackupTask(backupInterval, removeBackups, toKeep);
+                    } else {
+						try {
+							FileUtils.backupPurge(file, toKeep);
+						} catch (IOException e) {
+							Skript.error("Variables backup wipe failed: " + e.getLocalizedMessage());
+						}
+					}
```

Comment:
Other databases can require files aswell but not all databases use CSV extension. Can this go within the FlatFileStorage class? You can use the `load_i` method within the FlatFileStorage to get the same access to the ConfigurationSection